--- include/polarssl/config.h	2010-12-26 16:51:17.000000000 +0200
+++ include/polarssl/config.h	2011-02-24 22:10:48.043856002 +0200
@@ -78,7 +78,7 @@
 /*
  * Enable the checkup functions (*_self_test).
  */
-#define POLARSSL_SELF_TEST
+//#define POLARSSL_SELF_TEST
 
 /*
  * Enable run-time version information functions
--- library/rsa.c	2010-12-26 16:51:17.000000000 +0200
+++ library/rsa.c	2011-03-02 21:18:43.486888009 +0200
@@ -252,7 +252,7 @@
         return( POLARSSL_ERR_RSA_BAD_INPUT_DATA );
     }
 
-#if 0
+#if 1
     MPI_CHK( mpi_exp_mod( &T, &T, &ctx->D, &ctx->N, &ctx->RN ) );
 #else
     /*
